pkg/admissioncontroller/apis/config/v1alpha1/types.go

```diff
@@ -36,6 +36,10 @@ type AdmissionControllerConfiguration struct {
 	// Debugging holds configuration for Debugging related features.
 	// +optional
 	Debugging *componentbaseconfigv1alpha1.DebuggingConfiguration `json:"debugging,omitempty"`
+	// AllowTokenFile specifies if "tokenFile" field is allowed to be used in the kubeconfig file
+	// used for connecting to the Garden cluster.
```

Comment:
No hard opinion, but if you are pushing some other changes, you might add this, too.
```suggestion
	// used for connecting to the Garden cluster. Useful when projected tokens are used.
```

Reply:
This should rather be part of `GardenClientConnection` to make it clear. This can very fast become ambiguous.